app/controllers/api/account.php

```diff
@@ -540,7 +540,11 @@
 
         Authorization::setRole('user:' . $user->getId());
 
-        $dbForProject->updateDocument('users', $user->getId(), $user);
+        try {
+            $dbForProject->updateDocument('users', $user->getId(), $user);
```

Comment:
Shouldn't we expect this to be the case when updating a document? How is this error even possible?

Reply:
1. You sign in with OAuth
2. You sign out
3. You create an anonymous session
4. You use OAuth to claim your anonymous account

It will try to update your current anonymous account with a new email from OAuth. This ends up with duplicate error since there already is account with this email.